scripts/in_container/bin/run_tmux

```diff
@@ -55,16 +55,21 @@ tmux send-keys -t 'Main' 'bash' C-m 'clear' C-m
 
 tmux split-window -v
 tmux select-pane -t 1
-tmux send-keys 'opentelemetry-instrument airflow scheduler' C-m
+tmux send-keys '${COMMAND_PREFIX} airflow scheduler' C-m
 
 tmux split-window -h
 tmux select-pane -t 2
-tmux send-keys 'opentelemetry-instrument airflow webserver' C-m
+tmux send-keys '${COMMAND_PREFIX} airflow webserver' C-m
```

Comment:
Here we also want to add `--opentelemetry" if "jeager" is enabled. Similarly to the PREFIX below.

Reply:
ok